resource/speedometer.ui

```diff
@@ -6,10 +6,22 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>479</width>
-    <height>509</height>
+    <width>500</width>
+    <height>550</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>500</width>
```

Comment:
Is is possible to set here the value of the previous `<width>` field instead of hardcoding it?

Reply:
There are some values already hardcoded in the code to set the size of the widget https://github.com/ToyotaResearchInstitute/gauges2/blob/main/src/rqt_gauges_2/speedometer_gauge.py#L126-L127